runtime/freelist.c

```diff
@@ -879,7 +879,7 @@ static int bf_small_map = 0;
 /* Small free blocks have only one pointer to the next block.
    Large free blocks have 5 fields:
    tree fields:
-     - node flag
+     - isnode flag
      - left son
      - right son
```

Comment:
While here, out of curiosity of @damiendoligez, why is "son" used? FWIW is the more neutral "child" still appropriate?

Reply:
I guess it's a cultural thing. In French, we have no word for "sibling" so for trees instead of child/sibling we still use *fils*/*frère*. Then I naturally translate into english as son/brother. If you want to change these comments I won't object.

Reply:
Ah, I thought that might be it, thanks!

Reply:
Should I rename "son" to "child" then or is it ok like this?

Reply:
I think it’s slightly clearer as `child`, so yes, please